### PR TITLE
Typo-fixes for "triad-roles-and-responsibilites.md"

### PR DESCRIPTION
**Pull request by erzhtor**

### CODE REVIEW

File: content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+## Roles and Responsibilities
 
 ### Product Strategy
 
 #### Performing research to inform strategy
 
 - **Accountable**: PM
 - **Responsible**: PM
-- **Expectations**: The PM conducts market, customer and quantitative research and synthesizes this into insights that the team can understand.
+- **Expectations**: The PM conducts market, customer, and quantitative research and synthesizes this into insights that the team can understand.
```
Comment (ryankscott) [inline content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md:17]: :( looks like we prefer [Oxford commas](https://handbook.sourcegraph.com/company-info-and-process/communication/content_guidelines/style_and_mechanics/#commas=)

File: content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md [hunk 2]
```diff
@@ -124,31 +124,31 @@
...
 - **Responsible**: EM / Team
-- **Expectations**: PM is accountable for making sure the teams iterations are aligned with high level goals
+- **Expectations**: PM is accountable for making sure the teams iterations are aligned with high-level goals
 
 #### Providing guidance on prioritization
 
 - **Accountable**: PM
 - **Responsible**: Team
-- **Expectations**: PM is accountable for making sure we align with high level company goals, EM is expected to work closely with PM to ensure we make progress towards it and inform on technical details. The team should be proposing issues and priorities for an iteration.
+- **Expectations**: PM is accountable for ensuring we align with high-level company goals. EM is expected to work closely with PM to ensure we progress towards it and inform on technical details. The team should be proposing issues and priorities for an iteration.
```
Comment (ryankscott) [inline content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md]: I think we need to say "The EM is expected...."

File: content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md [hunk 3]
```diff
@@ -52,27 +52,27 @@
...
+- **Expectations**: The team can and should create issues based on the feedback coming in through any channel if it is not yet documented. The EM ensures that feedback is being captured in Github.
 
 ### Problem discovery
 
 #### Categorizing feedback
 
 - **Accountable**: PM
 - **Responsible**: PM / Team
-- **Expectations**: The PM is accountable for ensuring that feedback is acquired, shared, stored and categorized in a way that can be used to inform future decisions. This includes storing it in ProductBoard, adding any associated labels, features and ensuring everyone knows how to do this. The team should feel empowered to add any feedback they encounter from any sources they see.
+- **Expectations**: The PM is accountable for ensuring that feedback is acquired, shared, stored, and categorized in a way that can be used to inform future decisions. This includes storing it in ProductBoard, adding any associated labels, features, and ensuring everyone know how to do this. The team should feel empowered to add any feedback they encounter from any sources they see.
```
Comment (ryankscott) [inline content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md]: That should be "knows"

File: content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md [hunk 4]
```diff
@@ -124,31 +124,31 @@
...
 - **Responsible**: Team
-- **Expectations**: PM is accountable for making sure we align with high level company goals, EM is expected to work closely with PM to ensure we make progress towards it and inform on technical details. The team should be proposing issues and priorities for an iteration.
+- **Expectations**: PM is accountable for ensuring we align with high-level company goals. EM is expected to work closely with PM to ensure we progress towards it and inform on technical details. The team should be proposing issues and priorities for an iteration.
 
 #### Capacity planning
 
 - **Accountable**: EM
 - **Responsible**: Team
-- **Expectations**: EMs are accountable from making sure we balance the roadmap items and passive items (backlog/tech debt) with engineering capacity
+- **Expectations**: EMs are accountable for making sure we balance the roadmap items and passive items (backlog/tech debt) with engineering capacity
```
Comment (ryankscott) [inline content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md]: A side note, we should probably say "The EM ...."

File: content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md [hunk 5]
```diff
@@ -124,31 +124,31 @@
...
 
 #### Ticket refinement (scoping, measures of success)
 
 - **Accountable**: EM
 - **Responsible**: PM
-- **Expectations**: EMs are accountable for making sure the GitHub issues have all the required information for engineers to pick up and get started.
+- **Expectations**: EMs are accountable for ensuring the GitHub issues have all the required information for engineers to pick up and get started.
```
Comment (ryankscott) [inline content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md]: Similarly should be "The EM..." 
Also should be "ensuring that GitHub issues..."

File: content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md [hunk 6]
```diff
@@ -124,31 +124,31 @@
...
 
 #### Dependency management / logistics
 
 - **Accountable**: PM
 - **Responsible**: EM
-- **Expectations**: On simple cross team engineering dependencies, EMs working with other EMs generally work just fine. On larger, roadmap impacting items, the coordination efforts should be led by the PMs as they are accountable from the corresponding teams roadmap.
+- **Expectations**: On simple cross-team engineering dependencies, EMs working with other EMs generally work just fine. On larger roadmap impacting items, the PMs should lead the coordination efforts as they are accountable for the corresponding teams' roadmap.
```
Comment (ryankscott) [inline content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md]: Need to move to singular PM and EM, not plurals.

**Review by ryankscott — CHANGES_REQUESTED** [review]
Thanks for the review! Just left a few comments mostly unrelated to your changes but the original.

File: content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md (hunk 1 above)
Comment (a-bergevin) [reply content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md:17]: I myself love the Oxford comma, but I know it's controversial.

File: content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md (hunk 3 above)
Comment (a-bergevin) [reply content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md]: Agreed, this one should stay `knows`

File: content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md (hunk 1 above)
Comment (erzhtor) [reply content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md:17]: Yeah, I guess our corporate Grammarly was configured to use Oxford commas. 
Do you want me to revert all comma changes and leave just typo/rewording fixes? 

P.S: I don't have strong preferences on this)

Comment (ryankscott) [reply content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md:17]: Nope the Oxford comma is in our style guide so keep it.

**Review by ryankscott — APPROVED** [review]

**Pull request merged**